app/views/carts/show.html.erb

```diff
@@ -1,35 +1,34 @@
-<h1>Your Cart</h1>
+<h1>Pending Loans</h1>
 <%= render "shared/back_button" %>
 
 <div id="cart_projects">
   <table class='table table-bordered'>
     <thead>
       <tr class="warning">
-        <th data-field='Project Name'>Project</th>
-        <th data-field='Description'>Description</th>
-        <th data-field='Price'>Price</th>
-        <th data-field='Quantity'>Quantity</th>
-        <th data-field='Subtotal'>Subtotal</th>
+        <th data-field='Organization Name'>Organization Name</th>
+        <th data-field='Purpose'>Purpose</th>
+        <th data-field='Amount'>Amount</th>
         <th></th>
       </tr>
     </thead>
-    <% if @cart.nil? %>
-      <tr><td colspan='6'>Your Cart Is Empty</td></tr>
+    <% if @pending_loans.empty? %>
+      <tr><td colspan='6'>You Have No Pending Loans</td></tr>
     <% else %>
-      <% @cart.projects.each do |project, quantity| %>
+      <% @pending_loans.each do |project, loan_amount| %>
         <tr id=<%= "project_#{project.id}" %>>
+          <td><%= project.tenant.organization %></td>
           <td><%= link_to project.title, project_path(project) %></td>
-          <td><%= project.description %></td>
-          <td><%= number_to_currency project.price / 100 %></td>
-          <td id=<%= "quantity_#{project.id}" %>><%= quantity %></td>
-          <td><%= number_to_currency project.price.to_i * quantity.to_i / 100 %></td>
+          <td id=<%= "quantity_#{project.id}" %>><%= loan_amount %></td>
           <td>
-
-            <%= form_for :update_project_quantity, url: update_project_quantity_path do |f| %>
-              <%= f.select 'quantity', options_for_select((1..10)) %>
+            <%= form_for :update_pending_loan_amount, url: update_project_amount_path do |f| %>
+              <%= f.select(:loan_amount) do %>
+                <% [["$25", 25], ["$50", 50], ["$100", 100], ["$500", 500]].each do |amount| %>
```

Comment:
I thought we were just going to let people fully fund or not each loan? Obviously the add $25, $50, option is great :+1: , but can we add it to a waffle ticket instead? It seems like a lot more work and probably requires some DB restructuring, more logic throughout, etc.